CMakeLists.txt

```diff
@@ -76,6 +84,14 @@ if(NOT MSVC AND NOT ANDROID)
     )
     string(REPLACE ";" " " SUPPORTED_FLAGS "${SUPPORTED_FLAGS}")
     string(APPEND CMAKE_CXX_FLAGS " ${SUPPORTED_FLAGS}")
+    if (FACELIFT_ENABLE_GCOV)
+        check_supported_compiler_flags(SUPPORTED_FLAGS_GCOV
+            -fprofile-arcs
+            -ftest-coverage
+        )
+    endif()
+    string(REPLACE ";" " " SUPPORTED_FLAGS_GCOV "${SUPPORTED_FLAGS_GCOV}")
```

Comment:
do not duplicate code

Reply:
sure, but my statement is relevant to any future change. Code duplication should be avoided as much as possible.

Reply:
Code duplication unacceptable/forbidden in any case. This is not needed to discuss.